server/src/test/java/org/eclipse/lsp/cobol/usecases/TestEnterCompilerDirective.java

```diff
@@ -50,9 +55,14 @@ void testErrorOnEnterCompilerDirective_whenLanguage() {
         ImmutableMap.of(
             "1",
             new Diagnostic(
-                new Range(new Position(1, 7), new Position(1, 14)),
+                new Range(new Position(1, 7), new Position(1, 23)),
```

Comment:
Why does it point to a position that is further than the line end?

Reply:
Corrected. 
Problem:
DOT_FS was consuming till the next line. So for below code
`       Identification Division.
       ENTER .
       Program-Id. control-dir.`

mapping token had below token for enter stmt.
`ENTER .
       `